readme.md

```diff
@@ -113,6 +113,16 @@ runtime is automatic (`string`, default: `'react'`).
 Comment: `@jsxImportSource theSource`.
 Note that `/jsx-runtime` is appended to this provided source.
 
+##### `options.development`
+
+If the automatic runtime is used, this compiles JSX into automatic runtime
+development mode (`boolean`, default: `false`).
```

Comment:
```suggestion
Add location info on where a component originated from (`boolean`, default:
`false`).
This helps debugging but adds a lot of code that you don’t want in production.
Only used when `filePath` is and in the automatic runtime.
```

Btw: does React still show somewhat useful info without a `filePath`? Maybe we could still do something with line/column without and lift the restriction? Maybe more confusing but potentially we could default to `'<source.js>'` or so even?

Reply:
This is what React Devtools shows if `filePath` is missing.

![image](https://user-images.githubusercontent.com/779047/168901758-fe886704-ec3c-4e7f-a5cf-5f8ff1a9bcaf.png)

So let’s not add it in this case.

If positional information is missing, it’s rendered as `undefined`, which I think is ok.

Reply:
So it doesn’t show anything?
And what if we use `'<source.js>'`

Reply:
My screenshot was a bit unclear. This error view replaces what would normally be the part that display component details.

Anyway, showing `<source.js>` looks fine. :+1:

Reply:
Now I’m interested whether you have a screenshot that shows this, because I haven’t seen it yet!
Glad a fake name is somewhat useful!

Reply:
Sure!

I created the following Next.js page in [recma-nextjs-static-props](https://github.com/remcohaszing/recma-nextjs-static-props), then tweaked the source parameters:

```jsx
import { jsxDEV } from 'react/jsx-dev-runtime'

export default function Foo() {
  return jsxDEV(
    'p',
    { children: 'Hello world!' },
    'key',
    false,
    {
      fileName: '/home/remco/Projects/recma-nextjs-static-props/pages/foo.jsx',
      lineNumber: 35,
      columnNumber: 42,
    },
    this,
  )
}
```

Using the [React devtools chrome extension](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi), it looks like this:

![Complete information](https://user-images.githubusercontent.com/779047/168993904-3e68c194-9abe-40a2-84c1-15f2717f8795.png)

The source copy button copies `/home/remco/Projects/recma-nextjs-static-props/pages/foo.jsx:35` to the clipboard. This format can be copied directly into the VSCode file search for example

The _Log this component data to the console button_ logs the following positional data to the console:

![Component data logged to console](https://user-images.githubusercontent.com/779047/168994578-a570fe20-9eb3-4ed1-8464-ec2f5b6a66ab.png)

With removed positional information, it looks like this:

![missing positional information](https://user-images.githubusercontent.com/779047/168994691-75ddd876-c14f-48c3-8684-e158c791491c.png)

With a removed filename, it looks like this:

![missing filename](https://user-images.githubusercontent.com/779047/168994782-1ee0b5e9-2e3a-4799-8a03-70147a3ec74b.png)

With `source` set to undefined, it looks like this:

![`source` undefined](https://user-images.githubusercontent.com/779047/168995057-05c1d8da-e05c-4446-be34-04d9f8f96096.png)

And just for you, a screenshot with `fileName` set to `<source.js>`. :smile: 

![image](https://user-images.githubusercontent.com/779047/168996415-e4813639-57d5-4dbe-8c8a-6d88a0525105.png)